src/TortoiseProc/GitLogListBase.cpp

```diff
@@ -95,8 +95,16 @@ CGitLogListBase::CGitLogListBase():CHintListCtrl()
 	m_bFilterWithRegex = !!CRegDWORD(_T("Software\\TortoiseGit\\UseRegexFilter"), FALSE);
 	m_bFilterCaseSensitively = !!CRegDWORD(_T("Software\\TortoiseGit\\FilterCaseSensitively"), FALSE);
 
-	m_From = -1;
-	m_To = -1;
+	m_Filter.m_NumberOfLogsScale = (DWORD)CRegDWORD(_T("Software\\TortoiseGit\\LogDialog\\NumberOfLogsScale"), -1);
+	m_Filter.m_NumberOfLogs = (DWORD)CRegDWORD(_T("Software\\TortoiseGit\\LogDialog\\NumberOfLogs"), 0);

```

Comment:
Did the user requested to have those at the same this? E.g. last 3 weeks, last 5 months, 6 years to select from? - I thought our users just wanted to be able to set a concrete limit "5 years" OR "3 months" instead of last 400 commits.